import-export-cli/integration/testutils/api_testUtils.go

```diff
@@ -157,6 +157,7 @@ func exportAPI(t *testing.T, name string, version string, provider string, env s
 
 	t.Cleanup(func() {
 		base.RemoveAPIArchive(t, getEnvAPIExportPath(env), name, version)
+		base.WaitForIndexing()
```

Comment:
No need to wait here because we are only removing the export archive file and nothing related to indexing is taking place in APIM side.